extension/src/status.ts

```diff
@@ -91,39 +91,23 @@ export class Status extends Disposable {
       return
     }
 
-    if (this.available) {
-      return {
-        command: RegisteredCommands.EXTENSION_SETUP_WORKSPACE,
-        title: Title.SETUP_WORKSPACE
-      }
-    }
-
     return {
       command: RegisteredCommands.SETUP_SHOW,
       title: Title.SHOW_SETUP
     }
   }
 
-  private getEnvDetails() {
+  private getEnvIndicator() {
     const dvcPath = this.config.getCliPath()
     const pythonBinPath = this.config.getPythonBinPath()
     if (dvcPath || !pythonBinPath) {
-      return {
-        indicator: '(Global)',
```

Comment:
For now, I kept the indicator and removed the tooltip entirely. Other options include removing the indicator as well, adding a more static tooltip, or just keeping things the way they were originally.

Reply:
[Q] Outside of not having to maintain two different sets of text what is the benefit of removing the tooltip?

Reply:
> [Q] Outside of not having to maintain two different sets of text what is the benefit of removing the tooltip?

I thought that is what was requested since you can now just click on it, taking you to DVC CLI information in Setup. But I probably misunderstood something in a meeting or a comment. I can go ahead and add it back!

Reply:
Added tooltip back for now!